test/functional-portable/datastoresrp/noncloud/resources/testdata/datastoresrp-resources-redis-manual.bicep

```diff
@@ -1,5 +1,5 @@
 extension radius
-
+git config --global core.autocrlf true
```

Comment:
was this intended?

Reply:
no :'(

Reply:
<img width="349" alt="image" src="https://github.com/user-attachments/assets/a56fefac-22af-440d-8064-cb2a9ea60336">

Indeed 😁